src/modules/Resources/TagoCores.ts

```diff
@@ -215,7 +215,7 @@ class TagoCores extends TagoIOModule<GenericModuleParams> {
       params: { tcore: tagoCoreID },
     });
 
-    result = dateParser(result, ["last_input", "last_output", "updated_at", "created_at", "inspected_at"]);
+    result = dateParser(result, ["last_input", "updated_at", "created_at"]);
```

Comment:
`last_output`, `inspected_at` and `last_retention` are not returned by Device info route, so they were removed.